tests/test_depositor_bot.py

```diff
@@ -0,0 +1,189 @@
+        setup_web3_deposit_fixtures_with_high_gas,
+        depositor_bot,
+        remove_sleep,
+        setup_ping_message_to_kafka,
+        setup_deposit_message_to_kafka,
+):
+    caplog.set_level(logging.INFO)
+    depositor_bot.run_cycle()
+
+    assert find_log_message(caplog, depositor_bot.GAS_FEE_HIGHER_THAN_RECOMMENDED)
+    record = find_log_message(caplog, 'Issues found.')
+    assert record
+    assert record.msg['value'] == [depositor_bot.GAS_FEE_HIGHER_THAN_RECOMMENDED]
+    assert not find_log_message(caplog, 'No issues found. Try to deposit.')
+
+
+def test_deposit_issues__security_check(
+    caplog,
+    setup_web3_deposit_fixtures_prohibits_the_deposit,
+    depositor_bot,
+    remove_sleep,
+    setup_ping_message_to_kafka,
+    setup_deposit_message_to_kafka,
+):
+    caplog.set_level(logging.INFO)
+    depositor_bot.run_cycle()
+
+    assert find_log_message(caplog, depositor_bot.DEPOSIT_SECURITY_ISSUE)
+    record = find_log_message(caplog, 'Issues found.')
+    assert record
+    assert record.msg['value'] == [depositor_bot.DEPOSIT_SECURITY_ISSUE]
+    assert not find_log_message(caplog, 'No issues found. Try to deposit.')
+
+
+def test_deposit_issues__buffered_ether(
+    caplog,
+    setup_web3_deposit_fixtures_not_enough_buffered_ether,
+    depositor_bot,
+    remove_sleep,
+    setup_ping_message_to_kafka,
+    setup_deposit_message_to_kafka,
+):
+    caplog.set_level(logging.INFO)
+    depositor_bot.run_cycle()
+
+    assert find_log_message(caplog, depositor_bot.LIDO_CONTRACT_HAS_NOT_ENOUGH_BUFFERED_ETHER)
+    record = find_log_message(caplog, 'Issues found.')
+    assert record
+    assert record.msg['value'] == [depositor_bot.LIDO_CONTRACT_HAS_NOT_ENOUGH_BUFFERED_ETHER]
+    assert not find_log_message(caplog, 'No issues found. Try to deposit.')
+
+
+def test_deposit_issues__enough_signs(
+        caplog,
+        setup_web3_deposit_fixtures,
+        depositor_bot,
+        remove_sleep,
+        setup_ping_message_to_kafka,
+):
+    caplog.set_level(logging.INFO)
+    depositor_bot.run_cycle()
+
+    assert find_log_message(caplog, depositor_bot.QUORUM_IS_NOT_READY)
+    record = find_log_message(caplog, 'Issues found.')
+    assert record
+    assert record.msg['value'] == [depositor_bot.QUORUM_IS_NOT_READY]
+    assert not find_log_message(caplog, 'No issues found. Try to deposit.')
+
+
+def test_deposit_issues__no_free_keys(
+    caplog,
+    setup_web3_deposit_fixtures_no_free_keys,
+    depositor_bot,
+    remove_sleep,
+    setup_ping_message_to_kafka,
+    setup_deposit_message_to_kafka,
+):
+    caplog.set_level(logging.INFO)
+    depositor_bot.run_cycle()
+
+    assert find_log_message(caplog, depositor_bot.LIDO_CONTRACT_HAS_NO_FREE_SUBMITTED_KEYS)
+    record = find_log_message(caplog, 'Issues found.')
+    assert record
+    assert record.msg['value'] == [depositor_bot.LIDO_CONTRACT_HAS_NO_FREE_SUBMITTED_KEYS]
+    assert not find_log_message(caplog, 'No issues found. Try to deposit.')
```

Comment:
Maybe we should put these messages ('Issues found.' and 'No issues found. Try to deposit.') in the constants as well as the problems?

Reply:
Yeah. Fixed